docs/advanced-topics/message-system/rpc-params.md

```diff
@@ -4,15 +4,69 @@ title: RPC Params
 ---
 import ImageSwitcher from '@site/src/ImageSwitcher.js';
 
-Both `ServerRpc` and `ClientRpc` methods can be configured either by `[ServerRpc]` and `[ClientRpc]` attributes at compile-time and `ServerRpcParams` and `ClientRpcParams` at runtime.
+`Rpc` methods are configured by the `[Rpc]` attribute at compile-time and `RpcParams` are configured at runtime. Likewise, the **legacy** `ServerRpc` and `ClientRpc` methods can be configured either by `[ServerRpc]` and `[ClientRpc]` attributes at compile-time and `ServerRpcParams` and `ClientRpcParams` at runtime.
```

Comment:
```suggestion
The `[Rpc]` attribute configures `Rpc` methods at compile-time and configures `RpcParams` at runtime. You can configure the **legacy** `ServerRpc` and `ClientRpc` methods in the following ways: 
* Use the `[ServerRpc]` and `[ClientRpc]` attributes at compile-time.
* Use `ServerRpcParams` and `ClientRpcParams` at runtime.
```

Reply:
This particular suggestion strikes me as a little off, because the first and second parts are saying the same thing: attribute at compile time and params at runtime. I feel like they should either both be presented as bullet points, or both in paragraph form.

There are three attributes: Rpc, ServerRpc, and ClientRpc. These are all compile-time configuration and all work identically. Each has an associated configuration override struct that can alter the configuration at runtime: RpcParams, ServerRpcParams, and ClientRpcParams, respectively.

Reply:
This does feel like a cleaner approach, thanks :) Updated suggestion to keep the info in the bullet points.

Reply:
Seems like the info about `[Rpc]` is misssing now, looks like it only mentions `[ServerRpc]` and `[ClientRpc]` now.

Reply:
Sorry about that omission, I've updated my suggestion. For the sake of getting this merged, if my suggestion is inaccurate, you can change it or leave it out. I will come back to these docs after release to review them properly and I don't want to delay you any longer!